search/search_quality/assessment_tool/result_view.cpp

```diff
@@ -0,0 +1,97 @@
+#include "search/search_quality/assessment_tool/result_view.hpp"
+
+#include "search/result.hpp"
+#include "search/search_quality/assessment_tool/helpers.hpp"
+
+#include <string>
+
+#include <QtWidgets/QHBoxLayout>
+#include <QtWidgets/QLabel>
+#include <QtWidgets/QRadioButton>
+#include <QtWidgets/QVBoxLayout>
+
+namespace
+{
+QLabel * CreateLabel(QWidget & parent)
+{
+  QLabel * label = new QLabel(&parent);
+  label->setSizePolicy(QSizePolicy::Preferred, QSizePolicy::Minimum);
+  label->setWordWrap(true);
+  return label;
+}
+
+void SetText(QLabel & label, std::string const & text) {
+  if (text.empty())
+  {
+    label.hide();
+    return;
+  }
+
+  label.setText(ToQString(text));
+  label.show();
+}
+}  // namespace
+
+ResultView::ResultView(search::Result const & result, QWidget & parent) : QWidget(&parent)
+{
+  Init();
+  SetContents(result);
+}
+
+void ResultView::SetRelevance(search::Sample::Result::Relevance relevance)
```

Comment:
As before - the type is used only once here. Where do you propose to put usings?